### ISSUE TITLE
Provide access to datatable/paginator variables in paginator template

### ISSUE THREAD

**Issue by Ketec** [primefaces/primeng#4481]
[ ] bug report => Search github for a similar issue or PR before submitting
[x ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35

**Current behaviour**
Static template with no variable access

**Expected behavior**
Access to various pagination related variables: 
[rows]="rows" [first]="first" [totalRecords]="totalRecords" [pageLinkSize]="pageLinks"

For example:
```
<ng-template pTemplate="paginatorLeft" let-totalRecords="totalRecords" let-first="first"  let-rows="rows">
{{totalRecords < 1 ? 0 : first + 1}} - {{totalRecords > 0 ? ((rows + first) <= totalRecords ? (rows + first) : totalRecords) : 0}} of {{totalRecords ? totalRecords : 0}}
</ng-template>
```

**Comment by haris-s** [comment]
Is this feature not available in 4.0.0 version?

**Comment by Ketec** [comment]
@haris-s unlikely, since it was added in 5.0.1
